dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -92,6 +92,10 @@ const METRICS_CONFIG = [
     title: "Node Disk",
     path: "/d-solo/rayDefaultDashboard/default-dashboard?orgId=1&theme=light&panelId=6",
   },
+  {
+    title: "Node Disk IO Speed",
```

Comment:
```suggestion
    title: "Node Disk IO",
```

(to be consistent with network below)